devolutions-gateway/src/recording.rs

```diff
@@ -676,3 +690,32 @@ async fn recording_manager_task(
 
     Ok(())
 }
+
+pub async fn remux(input_path: Utf8PathBuf) -> anyhow::Result<()> {
+    // CPU-intensive operation potentially lasting much more than 100ms.
+    tokio::task::spawn_blocking(move || remux_impl(input_path)).await??;
```

Comment:
issue: Check that Cadeau was loaded before using it.
```suggestion
    if cadeau::xmf::is_init() {
        // CPU-intensive operation potentially lasting much more than 100ms.
        tokio::task::spawn_blocking(move || remux_impl(input_path)).await??;
    }
```